### DISCUSSION TITLE
TL-IPC45AW-PLUS ONVIF added, password verification failed.

### DISCUSSION THREAD

**Discussion by love768952693** [koush/scrypted#999] [Q&A] [answered]
Tested that using ONVIF directly in Home Assistant can be added

When using scrypted to add, there is no image on the UI interface. Checking the log shows that the password is always wrong.

I also tried to calculate the plaintext password into md5 Hash Generator in the password item, but it was still wrong.

Plugin information: ONVIF Camera Plugin 0.0.123 @scrypted/onvif
scrypted Version 0.41.0

scrypted log
`rtsp://192.168.199.66:554/stream1 RTSP/1.0
[Rebroadcast Plugin]: CSeq: 3
[Rebroadcast Plugin]: User-Agent: Scrypted
[Rebroadcast Plugin]: Authorization: Basic Og==
[Rebroadcast Plugin]: 
[Rebroadcast Plugin]: mainStream prebuffer session ended with error Error: auth failed
[Rebroadcast Plugin]:     at t.RtspClient.request (/@scrypted/prebuffer-mixin/main.nodejs.js:1:91326)
[Rebroadcast Plugin]:     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[Rebroadcast Plugin]:     at async t.startRtspSession (/@scrypted/prebuffer-mixin/main.nodejs.js:1:143016)
[Rebroadcast Plugin]:     at async U.startPrebufferSession (/@scrypted/prebuffer-mixin/main.nodejs.js:1:116729)
[Rebroadcast Plugin]: TL-IPC45AW-PLUS restarting prebuffer session in 5 seconds`

**Comment by koush** [comment, marked as answer]
enable only digest auth per the camera setup guide
